DraggableCollectionView/Helpers/LSCollectionViewLayoutHelper.m

```diff
@@ -94,4 +105,33 @@ - (NSArray *)modifiedLayoutAttributesForElements:(NSArray *)elements
     return elements;
 }
 
+- (void)rearrangeIndexPathOfLayoutAttributesForElements:(NSArray *)elements
+{
+    if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 9.0)
+    {
+        NSComparator cmptr = ^(NSIndexPath *ele1, NSIndexPath *ele2){
+            
+            if (ele1.section < ele2.section)
+                return (NSComparisonResult)NSOrderedAscending;
+            
+            if (ele1.row < ele2.row)
+                return (NSComparisonResult)NSOrderedAscending;
+            
+            return (NSComparisonResult)NSOrderedDescending;
+        };
+        
+        NSMutableArray *indexPathArray = [NSMutableArray array];
+        
+        for (UICollectionViewLayoutAttributes *layoutAttributes in elements) {
+            [indexPathArray addObject:layoutAttributes.indexPath];
```

Comment:
if([layoutAttributes.representedElementKind isEqualToString:UICollectionElementKindSectionHeader] || [layoutAttributes.representedElementKind isEqualToString:UICollectionElementKindSectionFooter]) {
                continue;
            } 

Add this check at line 127. The issue is that the header and footer shouldn't have their indexPaths altered as they are always set to one thing even in a draggableCollectionView.